tests/Util/UtilTest.php

```diff
@@ -22,4 +22,11 @@ public function testSha256() {
         ));
         $this->assertEquals("YtbpYcrdbvk0RSVwTwENMzomS0LYtiItMwXhI5tohXs=", $signature);
     }
+
+    public function testFormatAmount() {
+        $amount = 15.02;
+        $currency = "EUR";
+        $formattedAmount = Util::formatAmount($amount, $currency);
+        $this->assertEquals(1502, $formattedAmount);
+    }
```

Comment:
shall we add 3 test cases for the 3 different options ?

Reply:
done